client/lib/shortcode/test/index.js

```diff
@@ -1,20 +1,18 @@
 /**
  * External depenendencies
  */
-var expect = require( 'chai' ).expect,
-	rewire = require( 'rewire' );
-
-/**
- * Internal dependencies
- */
-var Shortcode = rewire( '../' );
+var expect = require( 'chai' ).expect;
```

Comment:
We should just change this to `import` syntax, especially since it's the only dependency in the file.